src/main/java/cubicchunks/worldgen/generator/custom/CustomTerrainGenerator.java

```diff
@@ -206,6 +216,17 @@ private void generateTerrain(ICubePrimer cube, double[][][] input, int cubeX, in
 		}
 	}
 
+	/**
+	 * Retrieve the blockstate appropriate for the specified noise parameters
```

Comment:
This method is actually specifically for dirt/grass/surface block generation and nothing else